ern-composite-gen/src/createMetroConfig.ts

```diff
@@ -2,23 +2,29 @@ import fs from 'fs-extra';
 import path from 'path';
 import beautify from 'js-beautify';
 import os from 'os';
+import semver from 'semver';
 
 export async function createMetroConfig({
   cwd,
   projectRoot,
   blacklistRe,
   extraNodeModules,
   watchFolders,
+  reactNativeVersion,
 }: {
   cwd?: string;
   projectRoot?: string;
   blacklistRe?: RegExp[];
   extraNodeModules?: { [pkg: string]: string };
   watchFolders?: string[];
+  reactNativeVersion: string;
 }) {
+  const blackistSource = semver.gte(reactNativeVersion, '0.64.0')
```

Comment:
minor: repetitive block could go as a utility method. 

https://github.com/electrode-io/electrode-native/pull/1782/files#diff-c7e7027125e806eb119318c80986bcf7c23337bf85a72303eb9c8ef182f1398aR248-R253

Reply:
Yes you are totally right 👍  Guess I got lazy on this one.
Moved to a utility function.